api/src/main/java/jakarta/el/StaticFieldELResolver.java

```diff
@@ -80,7 +80,7 @@ public Object getValue(ELContext context, Object base, Object property) {
                 Field field = klass.getField(fieldName);
 
                 int mod = field.getModifiers();
-                if (Modifier.isPublic(mod) && Modifier.isStatic(mod)) {
+                if (isPublic(mod) && isStatic(mod)) {
                     return field.get(null);
```

Comment:
I'd prefer to see the static imports removed but I guess that falls under personal style preference. Consistent is certainly better that what we have now.

Reply:
Methods already are imported and used.